.github/workflows/build-and-test.yml

```diff
@@ -31,14 +31,6 @@ jobs:
       - name: Yarn Install
         run: yarn install --mode=skip-build && yarn allow-scripts
 
-      - name: Set Environment Variables
-        env:
-          INFURA_ID: ${{ secrets.INFURA_ID }}
-          API_KEY_ALCHEMY: ${{ secrets.API_KEY_ALCHEMY }}
-        run: |
-          echo "INFURA_ID=$INFURA_ID" > ./packages/dapp/.env
-          echo "API_KEY_ALCHEMY=$API_KEY_ALCHEMY" >> ./packages/dapp/.env
-
```

Comment:
I could not find these being used anywhere else in the codebase so I removed this step.